sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,560 @@
+    "    enable_online_store=False, \n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+ new_kms_key['KeyMetadata']['KeyId']\n",
+    ")\n",
+    "\n",
+    "orders_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False,\n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+new_kms_key['KeyMetadata']['KeyId']\n",
+    ")\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this example we create an online feature store that encrypts your data using your KMS key.\n",
+    "\n",
+    "**Note**: You will need to substitute your Account number in `arn:aws:kms:us-east-1:123456789012:key/` replacing `123456789012` with your Account number."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False, \n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+ new_kms_key['KeyMetadata']['KeyId']\n",
+    ")\n",
+    "\n",
+    "orders_feature_group.create(\n",
+    "    s3_uri=f\"s3://{s3_bucket_name}/{prefix}\",\n",
+    "    record_identifier_name=record_identifier_feature_name,\n",
+    "    event_time_feature_name=\"EventTime\",\n",
+    "    role_arn=role,\n",
+    "    enable_online_store=False,\n",
+    "    offline_store_kms_key_id = 'arn:aws:kms:us-east-1:123456789012:key/'+new_kms_key['KeyMetadata']['KeyId']\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to verify that your KMS key is being used to encrypt your data in your Online or Offline Feature Store? \n",
+    "\n",
+    "### Online Store Verification\n",
+    "To demonstrate that your data is being encrypted in your Online store, use your `kms` client from `boto3` to list the grants under your KMS key. It should show 'SageMakerFeatureStore-' and the name of your feature group you created and should list these operations under Operations:`['Decrypt','Encrypt','GenerateDataKey','ReEncryptFrom','ReEncryptTo','CreateGrant','RetireGrant','DescribeKey']`\n",
+    "\n",
+    "An alternative way for you to check that your data is encrypted in your Online store is to check [Cloud Trails](https://console.aws.amazon.com/cloudtrail/) and navigate to your account name. Once here, under General details you should see that SSE-KMS encryption is enabled and with your AWS KMS key shown below it. Below is a screenshot showing this: \n",
```

Comment:
"You can also make sure that your data is encrypted in your online store by checking [Cloud Trails](https://console.aws.amazon.com/cloudtrail/) and navigating to your account name. Under General details, check that SSE-KMS encryption is enabled with your AWS KMS key shown below it, as shown in the following screenshot."